src/c2pa/asset.rs

```diff
@@ -7,66 +7,108 @@ use crate::error::{Error, Result};
 use serde::{Deserialize, Serialize};
 use std::path::Path;
 
-/// Types of assets in the C2PA framework
+/// C2PA asset types for machine learning artifacts
+///
+/// These types follow the C2PA naming convention for different asset categories.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
 pub enum AssetType {
     // Datasets
+    /// Generic dataset
     #[serde(rename = "c2pa.types.dataset")]
     Dataset,
 
+    /// TensorFlow dataset format
     #[serde(rename = "c2pa.types.dataset.tensorflow")]
     DatasetTensorFlow,
 
+    /// PyTorch dataset format
     #[serde(rename = "c2pa.types.dataset.pytorch")]
     DatasetPyTorch,
 
+    /// ONNX dataset format
     #[serde(rename = "c2pa.types.dataset.onnx")]
     DatasetOnnx,
 
     // Models
+    /// Generic model
     #[serde(rename = "c2pa.types.model")]
     Model,
 
+    /// TensorFlow model
     #[serde(rename = "c2pa.types.model.tensorflow")]
     ModelTensorFlow,
 
+    /// PyTorch model
     #[serde(rename = "c2pa.types.model.pytorch")]
     ModelPyTorch,
 
+    /// ONNX model
     #[serde(rename = "c2pa.types.model.onnx")]
     ModelOnnx,
 
+    /// OpenVINO model
     #[serde(rename = "c2pa.types.model.openvino")]
     ModelOpenVino,
 
     // Software
+    /// Software/code artifact
     #[serde(rename = "c2pa.types.software")]
     Software,
 
+    /// Generator/tool artifact
     #[serde(rename = "c2pa.types.generator")]
     Generator,
 
     // Formats
+    /// NumPy array format
     #[serde(rename = "c2pa.types.format.numpy")]
     FormatNumpy,
```

Comment:
Sorry I didn't catch this earlier. Can we call this and the other two storage formats SerializationFormats, or something like this? I just want to distinguish from the actual dataset/model formats. Also, I think the 2.2 spec still just refers to these as "types" for some reason.
```suggestion
    // SerializationFormats
    /// NumPy array format
    #[serde(rename = "c2pa.types.numpy")]
    SerializationNumpy
```